React/Views/RCTShadowView.m

```diff
@@ -86,11 +86,46 @@ static void RCTProcessMetaProps(const float metaProps[META_PROP_COUNT], float st
   : 0;
 }
 
+/*
+ * Sets the measure function on the cssNode, when the
+ * aspectRatio rectangle is set. This helps calculate
+ * the height of the node based on width of parent and
```

Comment:
"...width of [the] parent..."

also can you adjust the line width of the comment to be 80 chars?